include/inviwo/core/processors/processor.h

```diff
@@ -194,13 +198,17 @@ class IVW_CORE_API Processor : public PropertyOwner,
      */
     void setIdentifier(std::string_view identifier);
     virtual const std::string& getIdentifier() const override;
+    NameDispatcherHandle onIdentifierChange(
```

Comment:
maybe add comment regarding arguments of std::function. Assuming "prev" and "new" identifier?